### PR TITLE
doc: add minutes for meeting 24 Apr 2024

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Review by MoLow — APPROVED** [review]

File: meetings/2024-04-24.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  * Benjamin, In terms of  the view of collaborators, reviews are most important to me, so making
+    sure we can retain maximum reviews may push against removing people
+  * Joyee, if we want to depend only on commit queue, we need people maintaining the commit
+    queue it is not stable enough to depend on.
+  * Matteo: Is the ability to run jenkins is it tied to the commit bit or the ability to write to the repo
+    * Richard, today there is a group which determines both write access and ability to launch CI
+      runs
+      * For request CI, it would be based on the same team
+  * Joyee, either you are a contributor or not. So if you are off having a kid, etc. Should not really
+    factor in whether you feel included etc. You have already opted out. If contributing to Node.js is still part of your life, you won’t be able to refrain yourself from doing at least one review, one commit in that year.
```
[the remark this answers is not in the document]
Comment (benjamingr) [reply meetings/2024-04-24.md:68]: Honestly I’m having a hard time with the exclusionary attitude as well in terms of what the project values have been historically.

[the remark this answers is not in the document]
Comment (mhdawson) [reply meetings/2024-04-24.md:68]: I'll apologize in advance if I paraphrased incorrectly in this case.

[the remark this answers is not in the document]
Comment (mhdawson) [reply meetings/2024-04-24.md:68]: Thinking about it further I'm going to remove that line from the minutes as regardless of what the discussion was I probably should have edited more appropriately.

Comment (mhdawson) [inline meetings/2024-04-24.md:68]:
```
```

**Pull request merged**